Tests/Brokerages/Bitfinex/BitfinexBrokerageExchangeInfoTests.cs

```diff
@@ -0,0 +1,40 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.ToolBox.BitfinexDownloader;
+using System;
+using System.Linq;
+
+namespace QuantConnect.Tests.Brokerages.Bitfinex
+{
+    [TestFixture]
+    public class BitfinexBrokerageExchangeInfoTests
+    {
+        [Test]
+        public void GetsExchangeInfo()
+        {
+            var downloader = new BitfinexExchangeInfoDownloader();
+            var tickers = downloader.Get().ToList();
+
+            Assert.IsTrue(tickers.Any());
+
+            foreach (var t in tickers)
+            {
+                Assert.IsTrue(t.StartsWith(Market.Bitfinex, StringComparison.OrdinalIgnoreCase));
+            }
```

Comment:
Think we could have a generic implementation of this unit tests for the different markets, where each market would just be a unit test case, and we can assert other things too: symbols are returned alphabetically, the expected column count with data